### PR TITLE
Add eslint, prettier and format the code in the tracks service. Closes #47

### LINKED ISSUE #47: add formatting and linting tools in Track Service

### PR DESCRIPTION
**Pull request by DanilByaliy**

### CODE REVIEW

File: services/tracks/src/app.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
-'use strict';
+'use strict'
```
Comment (KhrapkoVasyl) [inline services/tracks/src/app.js]: You must use a semicolon at the end of lines

**Review by KhrapkoVasyl — CHANGES_REQUESTED** [review]
You must use a semicolon at the end of lines

File: services/tracks/src/app.js (hunk 1 above)
Comment (KhrapkoVasyl) [reply services/tracks/src/app.js]: For the entire project

**Review by Bogdan-Zinovij — CHANGES_REQUESTED** [review]
Use semicolons in eol

**Review by KhrapkoVasyl — APPROVED** [review]

**Review by Bogdan-Zinovij — APPROVED** [review]

**Pull request merged**